### PR TITLE
Includes AboutUs V1.0

### PR DESCRIPTION
**Pull request by muserr**
AboutUs adoc file is attached.

### CODE REVIEW

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+We are a Chess team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
+
+== Project Team
+
+=== *Mohideen Imran Khan*
+image::mohideenik.png[width="150", align="left"]
+https://github.com/mohideenik[Knight Mohideen github]
+//[<<mohideenik#, portfolio>>]
+
+Role: Project Team Leader +
```
Comment (mohideenik) [inline docs/AboutUs.adoc]: Has been changed to Benjamin who got the majority votes

Comment (mohideenik) [reply docs/AboutUs.adoc]: @nishanthelango @limsiying @muserr

**Comment by muserr** [comment]
Have updated the roles for AboutUs

**Comment by muserr** [comment]
Have updated the AboutUs to V1.3, kindly review! Thanks!

**Pull request merged**